Makefile

```diff
@@ -49,3 +49,28 @@ coverage:
 coveralls: coverage
 	go get github.com/mattn/goveralls
 	goveralls -coverprofile=$(COVERAGE_FILE) -service=github
+
+DURATION ?= 3s
+COUNT ?= 5
+
+.PHONY: bench
+bench:
+	rm -rf bench
+	mkdir bench
+	go clean -testcache
+	go test -bench=. -benchmem -benchtime=${DURATION} -count=${COUNT} 2>&1 | tee bench/bench.txt
```

Comment:
Probably path to a file with results should be defined in a variable.
Although I'll not insist. It's up to you.

Reply:
Fixed.